fv3core/testing/validation.py

```diff
@@ -114,14 +122,22 @@ def _set_nans(self, tracers: Mapping[str, Quantity]):
     return SelectivelyValidatedTracerAdvection
 
 
-def get_compute_domain(
+def get_compute_domain_k_interfaces(
     instance,
 ) -> Tuple[Tuple[int, ...], Tuple[int, ...]]:
     origin = instance.grid.compute_origin()
     domain = instance.grid.domain_shape_compute(add=(0, 0, 1))
     return origin, domain
 
 
+def get_compute_domain_2d(
+    instance,
+) -> Tuple[Tuple[int, ...], Tuple[int, ...]]:
+    origin = (instance.grid.is_, instance.grid.js)
+    domain = (instance.grid.npx, instance.grid.npy)
+    return origin, domain
```

Comment:
Or maybe a better code-reuse way to write this function.
```suggestion
    return get_compute_domain_k_interfaces(instance)[:2]
```

Reply:
I don't like this as much as the other suggestion. Code reuse is good, but I like having each function be independent conceptually.

Reply:
I liked this suggestion more when the other function was called `get_compute_domain` and this one was called `get_compute_domain_2d`.